src/main/java/com/j256/simplemagic/entries/MagicMatcher.java

```diff
@@ -26,30 +26,30 @@
 	 * @return The object to be passed to {@link #isMatch(Object, Long, boolean, Object, MutableOffset, byte[])} or null
 	 *         if not enough bytes.
 	 */
-	public Object extractValueFromBytes(int offset, byte[] bytes, boolean required);
+	Object extractValueFromBytes(int offset, byte[] bytes, boolean required);
 
 	/**
 	 * Matches if the bytes match at a certain offset.
 	 * 
 	 * @return The extracted-value object, or null if no match.
 	 */
-	public Object isMatch(Object testValue, Long andValue, boolean unsignedType, Object extractedValue,
-			MutableOffset offset, byte[] bytes);
+	Object isMatch(Object testValue, Long andValue, boolean unsignedType, Object extractedValue,
+				   MutableOffset offset, byte[] bytes);
 
 	/**
 	 * Returns the string version of the extracted value.
 	 */
-	public void renderValue(StringBuilder sb, Object extractedValue, MagicFormatter formatter);
+	void renderValue(StringBuilder sb, Object extractedValue, MagicFormatter formatter);
 
 	/**
 	 * Return the starting bytes of the pattern or null if none.
 	 */
-	public byte[] getStartingBytes(Object testValue);
+	byte[] getStartingBytes(Object testValue);
 
 	/**
 	 * Offset which we can update.
 	 */
-	public static class MutableOffset {
```

Comment:
Do you know what `static` on an inner class does?  Please look this up in your java manual and revert this change.